checker/rules/session_rules.json

```diff
@@ -1,5 +1,5 @@
 {
   "session_level": {
-    "regexp": "^\\d{6}_\\d{3}_([a-zA-Z]{1})_([a-zA-Z]*)_([a-zA-Z-_0-9]*)"
+    "regexp": "^sess-\\d{6}_\\d{3}_([a-zA-Z]{1})_([a-zA-Z]*)_([a-zA-Z-_0-9]*)"
```

Comment:
at the end of the regexp, does "_0-9" implies that the underscore is authorized? if so, I would change it so that the underscore is NOT authorized in the final string...